realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -415,6 +415,19 @@ public void close() {
     }
 
     /**
+     * Check if the {@link io.realm.Realm} instance has already been closed.
```

Comment:
We shouldn't mention BaseRealm it is package protected and only really used as a placeholder for Code. So it is not even part of our JavaDoc